### PR TITLE
Bug Resolved and new test cases added

### PR DESCRIPTION
**Pull request by pant-manas**
Added new test cases in TestDrive.test.tsx file

Bugs resolved

https://abhirams.atlassian.net/browse/P3ECWFC-81

https://abhirams.atlassian.net/browse/P3ECWFC-80

https://abhirams.atlassian.net/browse/P3ECWFC-78

https://abhirams.atlassian.net/browse/P3ECWFC-77

https://abhirams.atlassian.net/browse/P3ECWFC-76

https://abhirams.atlassian.net/browse/P3ECWFC-75

https://abhirams.atlassian.net/browse/P3ECWFC-74

### CODE REVIEW

**Review by pashedin — APPROVED** [review]
Nice Work 👍 
scroll To (0,0) is the overdelivery that's well received 😉

**Comment by pant-manas** [comment]
> Nice Work 👍 scroll To (0,0) is the overdelivery that's well received 😉

Thanks @pashedin

**Comment by pashedin** [comment]
Kudos to Saleem as well for pointing that out 👏

File: src/pages/Careers/Careers.tsx [hunk 1]
```diff
@@ -36,7 +42,7 @@ function Careers() {
                   <Card.Title>E-MAIL</Card.Title>
                   <Card.Text>
                     <a href="mailto:carrers@xtremecars.com" className="formatLink">
```
Comment (pashedin) [inline src/pages/Careers/Careers.tsx]: Correct the casing for formatLink

Comment (pashedin) [reply src/pages/Careers/Careers.tsx]: it should be format-link, and check if the link effect to the css are getting applied as well

**Review by pashedin — CHANGES_REQUESTED** [review]
just 1 oopsie

File: src/pages/Careers/Careers.tsx (hunk 1 above)
Comment (pashedin) [reply src/pages/Careers/Careers.tsx]: change it to format-link here as well 😄

File: src/pages/Contact/Contact.tsx [hunk 2]
```diff
@@ -23,7 +29,7 @@ function Contact() {
                 <Card.Title className="contact--title">E-MAIL</Card.Title>
                 <Card.Text className="contact--text">
                   <a href="mailto:support@xtremecars.com" className="formatLink">
```
Comment (pashedin) [inline src/pages/Contact/Contact.tsx]: here too

File: src/pages/Careers/Careers.tsx (hunk 1 above)
Comment (pant-manas) [reply src/pages/Careers/Careers.tsx]: Okay @pashedin

**Review by pashedin — APPROVED** [review]

**Pull request merged**